scripts/publish-process/transform-bpk-imports.js

```diff
@@ -47,7 +47,9 @@ const updateImports = (file, dirs) =>
 // eslint-disable-next-line no-console
 console.log('Crunching Backpack import paths...');
 
-const dirs = execSync('ls packages | grep -v "react-version-test.js"')
+const dirs = execSync(
+  'ls packages | grep -v "react-version-test.js" | grep -v "bpk-mixins"',
```

Comment:
I wonder if we should have a file called `ignoredPackages` to act as a source of truth for packages to skip. Probably not worth it if it's only one though...

Reply:
I guess we could keep it in mind should we need to make it bigger and do more than one?